subprojects/libero-distribution/README.md

```diff
@@ -37,6 +37,72 @@ installed distribution will be under `build/install`
   * config files can be specified as a parameter
   * searches in `<projectdir>/.libero.yml`
   * searches in `~/.libero/<project-name>/.libero.yml`
+  
+## Cleaning local artifacts
+
+Libero now supports cleaning artifacts from the local repo prior to building the project
+
+Artifacts can be cleaned by providing the `project.maven.repo.clean` property with a list of comma separated group ids
+Any artifact located at or below that group id depth will be deleted
+
+## Using an alternate local maven repository
+
+An alternate local maven repo can be configured by providing the `project.maven.repo.local` property with a path to use for a local repository
```

Comment:
Sounds good, except for the fact that it would force a re-download of _everything_, which may be overkill. Do you think the new option @oconnormi to delete certain artifacts is good enough?